src/style.css

```diff
@@ -0,0 +1,40 @@
+* {
+  margin: 0;
+}
+    
+ul {
+  margin: 0;
+  padding: 0;
+}
+    
+.stars {
+  display: flex;
+  list-style: none;
+  margin-right: 0;
+}
+    
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
```

Comment:
you don't set any margin-left for the `stars__star` elements, do you need to reset it here?